Persons/src/Dancer.java

```diff
@@ -1,2 +1,8 @@
-public class Dancer {
-}
+class Dancer extends Person {
+
+    String groupName;
```

Comment:
why not private?

Reply:
I can make it private if necessary, but it wasn't requested